doc/client_lua_api.md

```diff
@@ -787,6 +787,8 @@ Call these functions only at load time!
       extra arguments and return the result
 * `fgettext(string, ...)` : returns string
     * same as fgettext_ne(), but calls core.formspec_escape before returning result
+* `minetest.pointed_thing_to_face_pos(placer, pointed_thing)`: returns a table
```

Comment:
I think "table" is correct. What's the content of the table is described in the next line. And the same term is used for other functions too

Just two examples from client_lua_api.md
```
* `minetest.deserialize(string)`: returns a table
* `minetest.serialize(table)`: returns a string
```